addons/website/views/snippets/s_tabs.xml

```diff
@@ -2,67 +2,62 @@
+                            <div class="oe_structure oe_empty"
+                                data-editor-message="DRAG BUILDING BLOCKS HERE"/>
+                        </div>
+                        <div class="tab-pane fade"
+                            id="nav_tabs_content_2"
+                            role="tabpanel"
+                            aria-labelledby="nav_tabs_link_2">
+                            <div class="oe_structure oe_empty"
+                                data-editor-message="DRAG BUILDING BLOCKS HERE"/>
                         </div>
-                        <div class="card-body tab-content">
-                            <div class="tab-pane fade show active"
-                                id="nav_tabs_content_1"
-                                role="tabpanel"
-                                aria-labelledby="nav_tabs_link_1">
-                                <h3>Details</h3>
-                                <p><b>Great stories have personality.</b> Consider telling a great story that provides personality. Writing a story with personality for potential clients will assists with making a relationship connection. This shows up in small quirks like word choices or phrases. Write from your point of view, not from someone else's experience.</p>
-                            </div>
-                            <div class="tab-pane fade"
-                                id="nav_tabs_content_2"
-                                role="tabpanel"
-                                aria-labelledby="nav_tabs_link_2">
-                                <h3>Shipping</h3>
-                                <p><b>Great stories are for everyone even when only written for just one person.</b> If you try to write with a wide general audience in mind, your story will ring false and be bland. No one will be interested. Write for one person. If it’s genuine for the one, it’s genuine for the rest.</p>
-                            </div>
-                            <div class="tab-pane fade"
-                                id="nav_tabs_content_3"
-                                role="tabpanel"
-                                aria-labelledby="nav_tabs_link_3">
-                                <h3>Review</h3>
-                                <blockquote>
-                                    <p>Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services.</p>
-                                    <footer>— Jane DOE, CEO of <b>MyCompany</b></footer>
-                                </blockquote>
-                            </div>
+                        <div class="tab-pane fade"
+                            id="nav_tabs_content_3"
+                            role="tabpanel"
+                            aria-labelledby="nav_tabs_link_3">
+                            <div class="oe_structure oe_empty"
+                                data-editor-message="DRAG BUILDING BLOCKS HERE"/>
```

Comment:
That message is supposed to be automatically added in JS. The code responsible for this addition should be called when a snippet is dropped in the page so that you don't have to c/c this here in the XML. Can you change that, in a dedicated commit? :)